daffodil-sapi/src/test/scala/org/apache/daffodil/example/TestScalaAPI.scala

```diff
@@ -887,7 +887,7 @@ class TestScalaAPI {
     try {
       res = dp.parse(input, outputter)
     } catch {
-      case e: Abort => {
+      case e: UsageException => {
```

Comment:
This is an API change, please remember to add a Deprecation/Compatibility section to your commit message.